provision/pool/pool_test.go

```diff
@@ -39,17 +42,6 @@ func (s *S) TearDownSuite(c *check.C) {
 	s.storage.Close()
 }
 
-func (s *S) SetUpTest(c *check.C) {
```

Comment:
Done, forgot that code from import cycle problems, also changed the rest to meet what was requested :D